src/rpdk/core/contract/suite/handler_read.py

```diff
@@ -0,0 +1,14 @@
+# fixture and parameter have the same name
+# pylint: disable=redefined-outer-name
+
+import pytest
+
+# WARNING: contract tests should use fully qualified imports to avoid issues
+# when being loaded by pytest
+from rpdk.core.contract.suite.handler_commons import test_read_failure_not_found
+
+
+@pytest.mark.read
+def contract_read_without_create(resource_client):
```

Comment:
you mean if resource been deleted? we cant read if it was not created

Reply:
Lets say we try to query an invalid primary identifier for a resource. It should fail with "Not Found"

Reply:
is that a possible scenario?